src/main/resources/changesets/changelog-consent-15.0.xml

```diff
@@ -5,61 +5,69 @@
+        <createTable tableName="accessrp">
             <column name="id" type="bigint" autoIncrement="true">
                 <constraints primaryKey="true" nullable="false"/>
             </column>
-            <column name="electionAccessId" type="bigint">
+            <column name="electionaccessid" type="bigint">
                 <constraints nullable="false"/>
             </column>
-            <column name="electionRPId" type="bigint">
+            <column name="electionrpid" type="bigint">
                 <constraints nullable="false"/>
             </column>
         </createTable>
     </changeSet>
 
     <changeSet author="vvicario" id="62">
-        <addForeignKeyConstraint baseColumnNames="electionAccessId" baseTableName="accessRp"
+        <validCheckSum>ANY</validCheckSum>
+        <addForeignKeyConstraint baseColumnNames="electionaccessid" baseTableName="accessrp"
                                  constraintName="fkElectionAcess" deferrable="false" initiallyDeferred="false"
-                                 onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="electionId"
+                                 onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="electionid"
                                  referencedTableName="election" referencesUniqueColumn="false"/>
     </changeSet>
 
     <changeSet author="vvicario" id="63">
-        <addForeignKeyConstraint baseColumnNames="electionRPId" baseTableName="accessRp"
+        <validCheckSum>ANY</validCheckSum>
+        <addForeignKeyConstraint baseColumnNames="electionrpid" baseTableName="accessrp"
                                  constraintName="fkElectionRP" deferrable="false" initiallyDeferred="false"
-                                 onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="electionId"
+                                 onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="electionid"
                                  referencedTableName="election" referencesUniqueColumn="false"/>
     </changeSet>
 
     <changeSet author="vvicario" id="64">
+        <validCheckSum>ANY</validCheckSum>
         <insert tableName="electiontype">
-            <column name="typeId" value="3"/>
+            <column name="typeid" value="3"/>
             <column name="type" value="RP"/>
         </insert>
     </changeSet>
 
     <changeSet author="vvicario" id="65">
+        <validCheckSum>ANY</validCheckSum>
         <addColumn tableName="vote">
-            <column name="type"  type="varchar(255)"></column>
+            <column name="type"  type="varchar(255)"/>
         </addColumn>
     </changeSet>
 
     <changeSet author="vvicario" id="66">
-        <sql dbms="hsql, mysql">
-            update vote set type = 'DAC' where isFinalAccessVote = 0;
-            update vote set type = 'FINAL' where isFinalAccessVote = 1;
+        <validCheckSum>ANY</validCheckSum>
+        <sql dbms="hsqldb, mysql">
```

Comment:
do you still need dbms="hsql, mysql" here?

Reply:
Yes, to restrict them from failing in postgres.